src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: scale, color, $transition-time;
```

Comment:
it should be like that 

<img width="439" alt="image" src="https://user-images.githubusercontent.com/102411130/227145573-d151d100-ed91-4e21-a9bd-a416ce3ee6bd.png">

```
transition: effect duration effect duration;